CONTRIBUTING.md

```diff
@@ -287,7 +287,7 @@ commits are signed-off and runs `black --check` and `pylint` to ensure the
 code will pass the lint check for python. You can enable it by running:
 
 ```bash
-git config core.hooksPath hooks
+pre-commit install -t pre-commit -t prepare-commit-msg
```

Comment:
Should we update the dev setup script to force-install this during script execution?

Reply:
Updated!